src/pudl/extract/epacems.py

```diff
@@ -159,9 +159,12 @@ def get_data_frame(self, partition: EpaCemsPartition) -> pd.DataFrame:
         dfs = []
         for month in range(1, 13):
             mf = partition.get_monthly_file(month)
-            with archive.open(str(mf.with_suffix(".zip")), "r") as mzip:
-                with ZipFile(mzip, "r").open(str(mf.with_suffix(".csv")), "r") as csv_file:
-                    dfs.append(self._csv_to_dataframe(csv_file))
+            try:
```

Comment:
This just allows missing monthly EPACEMS zip files to not exist and processing continues. Specifically for 2020 data where we do not have all months.